packages/account/src/Components/api-token/api-token-table.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { isMobile, formatDate } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+import ApiTokenContext from './api-token-context';
+import ApiTokenDeleteButton from './api-token-delete-button';
+import ApiTokenTableBodyRow from './api-token-table-row';
+import ApiTokenTableRowHeader from './api-token-table-row-header';
+import ApiTokenTableRowScopesCell from './api-token-table-row-scopes-cell';
+import ApiTokenTableRowTokenCell from './api-token-table-row-token-cell';
+import { TApiContext, TToken } from 'Types';
+
+const ApiTokenTable = () => {
+    const { api_tokens } = React.useContext<TApiContext>(ApiTokenContext);
+
+    const formatTokenScopes = (str: string) => {
+        const string = str || '';
```

Comment:
Do we need to pass `|| ""` here? because we are checking the type with typescript so the value of this shouldn't be empty